### ISSUE TITLE
List of mooneye-gb tests that don't pass

### ISSUE THREAD

**Issue by Kurayami7** [mgba-emu/mgba#2116]
As suggested. Forgive me if I make a mistake and include the wrong test with the wrong screenshot or vice versa. These all look similar. 

hblank_ly_scx_timing-GS

![hblank](https://user-images.githubusercontent.com/77198254/113493577-bb145d00-9523-11eb-8322-521428aaf856.png)

[hblank_ly_scx_timing-GS.zip](https://github.com/mgba-emu/mgba/files/6253685/hblank_ly_scx_timing-GS.zip)

intr_2_mode0_timing_sprites

![int2](https://user-images.githubusercontent.com/77198254/113493587-d717fe80-9523-11eb-9dfa-cdb9fa81e11d.png)

[intr_2_mode0_timing_sprites.zip](https://github.com/mgba-emu/mgba/files/6253689/intr_2_mode0_timing_sprites.zip)

lcdon_timing-GS

![lcd](https://user-images.githubusercontent.com/77198254/113493674-f2cfd480-9524-11eb-9a44-85b895ede975.PNG)

[lcdon_timing-GS.zip](https://github.com/mgba-emu/mgba/files/6253709/lcdon_timing-GS.zip)

lcdon_write_timing-GS

![write](https://user-images.githubusercontent.com/77198254/113493698-2d397180-9525-11eb-9ed2-8efa5e5f886d.PNG)

[lcdon_write_timing-GS.zip](https://github.com/mgba-emu/mgba/files/6253710/lcdon_write_timing-GS.zip)

boot_sclk_align-dmgABCmgb

![boot](https://user-images.githubusercontent.com/77198254/113493762-b81a6c00-9525-11eb-9ff0-af60a89e64d2.PNG)

[boot_sclk_align-dmgABCmgb.zip](https://github.com/mgba-emu/mgba/files/6253720/boot_sclk_align-dmgABCmgb.zip)

**Comment by endrift** [comment]
I hope you don't mind, but I'm gonna steal the OP and format it in a way that makes it easier to track.

Also you're missing a bunch.

**Comment by Kurayami7** [comment]
I'm just getting started lol

rapid_toggle

![rapid](https://user-images.githubusercontent.com/77198254/113493844-82c24e00-9526-11eb-985b-7f564fcfb2f9.PNG)

[rapid_toggle.zip](https://github.com/mgba-emu/mgba/files/6253728/rapid_toggle.zip)


boot_div2-S

![diva s](https://user-images.githubusercontent.com/77198254/113493927-3a576000-9527-11eb-809e-6db5485cd8f2.PNG)

[boot_div2-S.zip](https://github.com/mgba-emu/mgba/files/6253736/boot_div2-S.zip)


boot_div-dmg0

![diva0](https://user-images.githubusercontent.com/77198254/113493945-6541b400-9527-11eb-9024-ef8a1a6cdd0b.PNG)

[boot_div-dmg0.zip](https://github.com/mgba-emu/mgba/files/6253739/boot_div-dmg0.zip)


boot_div-dmgABCmgb

![boot dmc](https://user-images.githubusercontent.com/77198254/113493985-a33ed800-9527-11eb-95e9-f05f6b0de294.PNG)

[boot_div-dmgABCmgb.zip](https://github.com/mgba-emu/mgba/files/6253741/boot_div-dmgABCmgb.zip)


boot_div-S

![Capture](https://user-images.githubusercontent.com/77198254/113494024-016bbb00-9528-11eb-9344-367d6b75e421.PNG)

[boot_div-S.zip](https://github.com/mgba-emu/mgba/files/6253743/boot_div-S.zip)

**Comment by CasualPokePlayer** [comment]
boot-*letter* relies on the correct bootrom being used. It looks like you aren't actually using the correct bootrom for some of those.

**Comment by endrift** [comment]
@Kurayami7 I already put the full list in the OP

**Comment by Kurayami7** [comment]
Do you wanna just do this? It's a pain in the ass and other other guy said I already screwed up.

**Comment by endrift** [comment]
I handled it already so...yeah, that works heh.

But yes I'm not using the correct boot ROM since I want to get the timing right when booting without the boot ROM (excluding DMG0 which I might just skip) since you can specify the model at load time.

**Comment by CasualPokePlayer** [comment]
Note for the CGB timing tests, those are all CGB-DMG ROMs and CGB-DMG has different post bootrom timing than CGB, and CGB-DMG timing even varies based off other things in the header.

**Comment by endrift** [comment]
Yes, I'm aware of that already. It's a pain but I do emulate it to some extent.
